README.md

```diff
@@ -60,6 +61,20 @@ Connector runtimes provided by the Community:
 |  GitHub Dispatch          |            -           |      [GitHub](https://github.com/camunda-community-hub/camunda-8-connector-github-dispatch)           |   [README](https://github.com/camunda-community-hub/camunda-8-connector-github-dispatch/blob/main/README.md)            |   Apache 2.0      |         [![](https://img.shields.io/badge/Lifecycle-Proof%20of%20Concept-blueviolet)](https://github.com/Camunda-Community-Hub/community/blob/main/extension-lifecycle.md#proof-of-concept-)  |
 | OpenAI Moderation | - | [GitHub](https://github.com/MaxTru/camunda-connector-openai-moderation) | [README](https://github.com/MaxTru/camunda-connector-openai-moderation/blob/main/README.md) | MIT       |         [![](https://img.shields.io/badge/Lifecycle-Proof%20of%20Concept-blueviolet)](https://github.com/Camunda-Community-Hub/community/blob/main/extension-lifecycle.md#proof-of-concept-)  |
 
+# Adding connectors
+
+You are able to add your connector in just a few steps:
+- Be inspired with a brilliant idea!
+- [Implement it](https://docs.camunda.io/docs/components/connectors/custom-built-connectors/connector-sdk/#creating-a-custom-connector).
+- Test it with a [Self-Managed Camunda Platform](https://docs.camunda.io/docs/self-managed/about-self-managed/) instance.
+- Publish it at GitHub.
```

Comment:
as own repository?

Reply:
yes. We need to review the code first and make sure that at least essentials are present: readme, license, etc. Not all users have access to the `camunda-community-hub` in the beginning. We can transfer it later with the ticket if that is required.